sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -167,6 +167,30 @@ def test_lasso_zero():
     assert_almost_equal(clf.dual_gap_, 0)
 
 
+def test_enet_nonfinite_params():
+    # Check elasticnet throws ValueError when dealing with non-finite parameter
+    # values
+    X = np.array(
```

Comment:
We can randomly generate `X` and `y` with large enough values:

```python
    rng = np.random.RandomState(0)

    fmax = np.finfo(np.float64).max
    X = fmax * rng.uniform(size=(10, 2))
    y = rng.randint(0, 2, size=10)
```

REF: https://github.com/scikit-learn/scikit-learn/pull/22149/files#r780723219